components/NewsCard.jsx

```diff
@@ -7,7 +7,7 @@ import PropTypes from 'prop-types';
 
 const { height } = Dimensions.get('window');
 
-export default function NewsCard({ article, userID, scrollPercentage, articlesInView}) {
+export default function NewsCard({ article, scrollPercentage, articlesInView}) {
```

Comment:
what was userID used for before

Reply:
it is not used in the code anywhere of that file, so might as well keep it deleted